tox_poetry_installer/hooks.py

```diff
@@ -245,39 +177,18 @@ def tox_testenv_install_deps(venv: ToxVirtualEnv, action: ToxAction) -> Optional
             project_deps = []
             logger.info("Env does not install project package dependencies, skipping")
     except exceptions.ToxPoetryInstallerException as err:
-        venv.status = err.__class__.__name__
+        # tox_env.status = err.__class__.__name__
         logger.error(str(err))
-        return False
+        raise err
     except Exception as err:
-        venv.status = "InternalError"
+        # tox_env.status = "InternalError"
```

Comment:
This can be removed

Reply:
Removed

Reply:
Looks like this might've been lost in a rebase

Reply:
oops, re-removed